src/mesh/FloodingRouter.cpp

```diff
@@ -102,8 +102,7 @@ void FloodingRouter::perhapsRebroadcast(const meshtastic_MeshPacket *p)
 
                 LOG_INFO("Rebroadcast received floodmsg");
                 // Note: we are careful to resend using the original senders node id
-                // We are careful not to call our hooked version of send() - because we don't want to check this again
```

Comment:
Is this concern no longer valid? There don't seem to be corresponding changes to a local `send()`.

Reply:
I've gone through the history and it was added here: https://github.com/meshtastic/firmware/commit/ea24394110759a35f1aa141ea6809e74f4f677ab

It seems "this" was referring to `wasSeenRecently()`, which makes sense, because originally we didn't need to check this again. However, now we need to add our hop limit to the record, so it should actually be called again.

Reply:
Apparently I did not read the summary properly... looks like you already explained this there! Sorry for the spam...